index.bs

```diff
@@ -11302,6 +11306,147 @@ The [=remote end steps=] given |session| and |command parameters| are:
 </div>
 
 
+## The extensions Module ## {#module-extensions}
```

Comment:
Wonder if the module should not be called `WebExtensions` to avoid potential confusion with the `Extensions` section of the `Webdriver` spec.